action.yaml

```diff
@@ -68,3 +75,14 @@ runs:
     - name: Uninstall pre-commit hooks. # Don't break subsequent Git commands.
       run: pre-commit uninstall
       shell: bash
+    - name: Push a commit to main to bump version and update changelog.
+      if: >
```

Comment:
Why is there a ">" here...I'm not following the logic and wondering whether this is a typo or conditional logic that is just different than what I've seen previously

Reply:
It's [a quirk of YAML](https://yaml-multiline.info/).

Reply:
Huh.  Cool.  Well then I'll go ahead and approve this!